create a cluster pp (#45)
tidbcloud/dbaas-docs: create-tidb-cluster.md

1. Log in to TiDB Cloud.

>     Open <https://tidbcloud.com> in a browser and sign in with user name and password.

Reviewer: ```suggestion
    Open <https://tidbcloud.com> in a browser and sign in with your user name and password.
```

2. Select Cloud Provider.

    Click the cloud service provider icon on the left of the top navigation bar, and select the cloud provider. The existing options are Amazon Web Services and Google Cloud.

> 3. Access the **Create Cluster** page.

Reviewer: ```suggestion
3. Access the **Create a Cluster** page.
```

>     On the TiDB Cloud console’s homepage, click **Create Cluster**. The Create Cluster page is displayed.

Reviewer: I see that on the latest UI, it's "Create a Cluster", not "Create Cluster".
```suggestion
    On the TiDB Cloud console's homepage, click **Create a Cluster**. The Create a Cluster page is displayed.
```

    Click to select your cluster tier, which determines the throughput and performance of your cluster. You cannot change it after the cluster is created. See [Select Your Cluster Tier](select-cluster-tier.md) for details.

>     >**Note:**
>     >
>     >TiDB Cloud Public Preview Free Trial user only have access to t1.standard cluster tier.

Reviewer: ```suggestion
    > **Note:**
    >
    > TiDB Cloud Public Preview Free Trial users have access only to the t1.standard cluster tier.
```

7. Select the number of nodes.

    Set the number of your TiDB nodes and TiKV nodes respectively by clicking the plus or the minus sign. See [Size Your Cluster](size-your-cluster.md) for details.

>     >**Note:**
>     >
>     > TiDB Cloud Public Preview Free Trial user cannot change the number of TiDB and TiKV nodes.

Reviewer: ```suggestion
    > **Note:**
    >
    > TiDB Cloud Public Preview Free Trial users cannot change the number of TiDB and TiKV nodes.
```
